pkg/binding/bindingpolicy.go

```diff
@@ -87,11 +89,12 @@ func (c *Controller) handleBindingPolicy(ctx context.Context, obj runtime.Object
 
 		// set destinations and enqueue binding for syncing
 		c.bindingPolicyResolver.SetDestinations(bindingPolicy.GetName(), clusterSet)
-		c.logger.V(4).Info("enqueued Binding for syncing, while handling BindingPolicy", "name", bindingPolicy.Name)
+		logger.V(4).Info("enqueued Binding for syncing, while handling BindingPolicy", "name", bindingPolicy.Name)
```

Comment:
Missed this one, should be capitalized too.

Reply:
Done.